libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -79,7 +94,7 @@ export class CartService {
     const cart = await this.cartManager.createCart({
       interval: args.interval,
       offeringConfigId: args.offeringConfigId,
-      amount: 0,
+      amount: upcomingInvoice.totalAmount,
```

Comment:
I'm not sure this is the correct "total" from the upcoming invoice object.

Looking at [the Stripe docs](https://docs.stripe.com/api/invoices/object#invoice_object-total) invoice object `total` is the following.
> Total after discounts and taxes.

However, [the ticket FXA-8896](https://mozilla-hub.atlassian.net/browse/FXA-8896) is asking for the following
> Note: this should be irrespective of tax

Reply:
I don't believe "listAmount", which is Stripe Invoice `amount_due` is the right value either. [Stripe Invoice subtotal](https://docs.stripe.com/api/invoices/object#invoice_object-subtotal) seems like the right fit here.

> Total of all subscriptions, invoice items, and prorations on the invoice before any invoice level discount or exclusive tax is applied. Item discounts are already incorporated